src/likecounter.js

```diff
@@ -0,0 +1,10 @@
+export default function likeCounter(buttonId, likesData) {
+  const button = document.getElementById(buttonId);
+  if (button) {
+    const pokemonName = button.getAttribute('id');
+    const likeCount = likesData[pokemonName] || 0;
+    button.innerHTML = likeCount.toString();
+    return likeCount;
+  }
+  return 0;
+}
```

Comment:
- [ Required ] Good job so far guys, I think it would be better if you turn this function in to an `arrow function` the same way you've done for other functions in this project. Use of `ES6` syntax is one of the project requirements so let's fix this to satisfy the requirements.

## Example code.
```JavaScript
const likeCounter = (buttonId, likesData) => {
  // Your code goes here
}

export default likeCounter;
```

## Project wireframes
![Screenshot from 2023-07-08 01-03-16](https://github.com/Sonikak004/Js-capstone-project/assets/16392046/93f18a82-5ed1-4775-ad81-03bcb035b2cc)